### PR TITLE
Fix failing ODBC workflow

### PR DESCRIPTION
**Pull request by Yury-Fridlyand**
Signed-off-by: Yury-Fridlyand <yuryf@bitquilltech.com>

### Description
12.09.2022 GHA Windows env [was updated](https://github.com/actions/runner-images/commits/win19/20220911.1/images/win/Windows2019-Readme.md) and it caused ODBC workflow to crash.
See workflow runs [before](https://github.com/opensearch-project/sql/actions/runs/3040683379/jobs/4897012398) and [after](https://github.com/opensearch-project/sql/actions/runs/3040848729/jobs/4897361472) the update.
 
### Issues Resolved
Fixed failing ODBC workflow
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by acarbonetto — APPROVED** [review]
lgtm

File: sql-odbc/scripts/build_windows.ps1 [hunk 1]
```diff
@@ -21,9 +21,8 @@ $BUILD_DIR = "${WORKING_DIR}\build"
 # $BUILD_DIR = "${WORKING_DIR}\build\${CONFIGURATION}${BITNESS}"
 New-Item -Path $BUILD_DIR -ItemType Directory -Force | Out-Null
 
-$VCPKG_DIR = "${WORKING_DIR}/src/vcpkg"
-
-.\scripts\build_libcurl-vcpkg.ps1 $VCPKG_DIR $LIBCURL_WIN_ARCH
+$VCPKG_DIR = $Env:VCPKG_ROOT
```
Comment (acarbonetto) [inline sql-odbc/scripts/build_windows.ps1:24]: is $VCPKG_DIR used?

**Review by MaxKsyunz — APPROVED** [review]
Thanks for resolving it so quickly.

**Review by dai-chen — APPROVED** [review]
Thanks for your quick fix!

**Pull request merged**